command/genesis/genesis.go

```diff
@@ -235,6 +235,59 @@ func setFlags(cmd *cobra.Command) {
 			defaultBlockTimeDrift,
 			"configuration for block time drift value (in seconds)",
 		)
+
+		cmd.Flags().Uint64Var(
+			&params.checkpointInterval,
+			checkpointIntervalFlag,
+			defaultCheckpointInterval,
+			"number of blocks after which a new checkpoint is submitted",
```

Comment:
```
Checkpoint submission interval in blocks
```
Easier to read imo. Wdyt?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1519/commits/4a92ab07bf4ef242b2a664bf3fa2c71509a7289d